mender-convert-modify

```diff
@@ -196,7 +196,7 @@ EOF
     # folder, then any subfolders in versioned order.
     for candidate in work/rootfs/boot/dtbs $(find work/rootfs/boot/dtbs/ -maxdepth 1 -type d | sort -V -r); do
       if [ $(find $candidate -maxdepth 1 -name '*.dtb' | wc -l) -gt 0 ]; then
-        run_and_log_cmd "cp -r $candidate work/boot/dtb"
+        run_and_log_cmd "sudo cp -r $candidate work/boot/dtb"
```

Comment:
I'm surprised this is needed, as the boot partition is usually vfat, which doesn't need sudo to copy. But doesn't hurt.